### PR TITLE
chore: typo `suported` to `suportted`

### PR DESCRIPTION
**Pull request by Innei**

### CODE REVIEW

File: src/vs/workbench/services/languageDetection/browser/languageDetectionSimpleWorker.ts [hunk 1]
```diff
@@ -210,7 +210,7 @@ export class LanguageDetectionSimpleWorker implements ILanguageDetectionWorker {
 			case 'csv':
 			case 'toml':
 				// Other considerations for negativeConfidenceCorrection that
-				// aren't built in but suported by the model include:
```
Comment (offensive-vk) [inline src/vs/workbench/services/languageDetection/browser/languageDetectionSimpleWorker.ts:213]: Just some comment typo fix? Seriously

Comment (Innei) [reply src/vs/workbench/services/languageDetection/browser/languageDetectionSimpleWorker.ts:213]: Why not?

Comment (Simon-He95) [reply src/vs/workbench/services/languageDetection/browser/languageDetectionSimpleWorker.ts:213]: Why not？

Comment (offensive-vk) [reply src/vs/workbench/services/languageDetection/browser/languageDetectionSimpleWorker.ts:213]: I was expecting a typo in actual usable source code.

File: src/vs/workbench/services/languageDetection/browser/languageDetectionSimpleWorker.ts [hunk 2]
```diff
@@ -210,7 +210,7 @@ export class LanguageDetectionSimpleWorker implements ILanguageDetectionWorker {
...
+				// aren't built in but suportted by the model include:
```
Comment (sxzz) [inline src/vs/workbench/services/languageDetection/browser/languageDetectionSimpleWorker.ts:213]:
```suggestion
				// aren't built in but supported by the model include:
```

**Pull request closed without merging**